app/views/layouts/application.html.erb

```diff
@@ -8,9 +8,25 @@
 
     <%= stylesheet_link_tag "application", "data-turbo-track": "reload" %>
     <%= javascript_importmap_tags %>
+
+    <%= render partial: "shared/cdn_assets" %>
```

Comment:
Great job putting your CDN assets in a partial, this helps with adding new assets and not having everything inside your layout!